File: SampelApp/ios/App/App/Plugins/ContactsPlugin.m
//
//  ContactsPlugin.m
//  App
//
//  Created by Sebastian Vidrea on 13.05.2023.
//

#import <Capacitor/Capacitor.h>

CAP_PLUGIN(ContactsPlugin, "Contacts",
		   CAP_PLUGIN_METHOD(getContacts, CAPPluginReturnPromise);
)

File: SampelApp/ios/App/App/Plugins/ContactsPlugin.swift
//
//  ContactsPlugin.swift
//  App
//
//  Created by Sebastian Vidrea on 13.05.2023.
//

import Capacitor

@objc(ContactsPlugin)
public class ContactsPlugin: CAPPlugin {
	@objc func getContacts(_ call: CAPPluginCall) {
		call.resolve(
			[
				"results": [
					"Contact1",
					"Contact2"
				]
			]
		)
	}
}
